README.md

```diff
@@ -61,25 +61,28 @@ The manager looked at the engineer and was not happy.
 
 UPDATE: A fully encrypted laptop of our senior developer has been stolen from his bag while he was on a train, returning from a conference in late 2017. Spooky.
 
-### Encryption
+### Storage Encryption and Secure Work Environment
 
-All data related to the company and clients must be either stored in the cloud (Google Drive/Docs/Sheets in the company domain) or on encrypted media (encrypted volume or drive), unless the data in question is worthless or public.
+All data pertaining to the company and its clients must be either stored in the cloud (specifically, Google Drive/Docs/Sheets within the company domain) or on encrypted media (an encrypted volume or drive).
+This excludes data categorized as being of no value or already publicly available.
 
-EDIT: We are getting a lot of questions about who this applies to, so let's make it clear:
-everyone except:
+In addition, Developers roles (after 1nd stage of Trial period i.e. finishing Trial task#3) are required to set up a Secure Work Environment on their workstations.
+This is both for additional security and improved focus.
 
-- software developers in the first phase of the trial period (where they work on fake/mock data)
-- software developers who work exclusively on open-source projects (that only happened twice, though)
+Setting up Storage Encryption might initially appear daunting, but rest assured that it is fairly straightforward process.
+Our top recommendation is Full Disk Encryption (FDE), a method that encrypts the entire disk for comprehensive protection.
+Our second recommendation is Volume Encryption, a method that encrypts only the volume (/folder) where sensitive data is stored.
+Due to its ease of use, we recommend VeraCrypt by default, for which a simple tutorial can be found [here](https://github.com/reef-technologies/handbook/blob/master/docs/VeraCrypt.md).
+If you want to learn more about available options, please refer to the [Storage Encryption](docs/Storage_Encryption.md) document.
 
-So all staff, including virtual assistants, must have encrypted devices.
-Now that might sound scary, but these days setting up encryption is not that hard - in fact, instructions are provided in this document.
+If you plan to connect your mobile device to any work-related accounts, first make sure that you will be able to provide a documentation that your device is encrypted and is actively supported (i.e. receives regular security updates).
```

Comment:
we don't need it, since here we are putting that responsibility on the user; we are already doing that with their main OS. Phones are not any different in that regard.

How can one do that - like you say "it depends", but the easiest sanity check is - check the "Android security update" date. If its not from last 3months you are in trouble, as AFAIK Android releases them every month.

And then it is user responsibility to search for their phone exact configuration.
For me, it is Pixel, so my documentation would basically be: https://support.google.com/pixelphone/answer/4457705?hl=en#zippy=%2Cpixel-later-including-fold which assures me I will get security updates for next few years.